traveler-swift-core/TravelerKit/Models/DateRangeFilter.swift

```diff
@@ -12,9 +12,9 @@ import Foundation
  */
 public struct DateRangeFilter: Equatable {
     /// The range of availability dates with which the items should be filtered
-    public let range: ClosedRange<Date>
+    public let range: ClosedRange<Date>?
 
-    public init(range: ClosedRange<Date>) {
+    public init(range: ClosedRange<Date>?) {
```

Comment:
This is not correct. If you're initializing a DateRangeFilter you should always pass a date range. otherwise you run the risk of passing an empty date range filter. If you want users to NOT pass a date range then it should be optional in the query. In other words, if you give the users of the sdk the ability to instantiate models you have to force them to instantiate a complete model to avoid unexpected behavior.

Reply:
If the user clears the date selection thats the WLA responsibility to handle not SDK.

Reply:
That makes sense. I'll handle that in WLA and close this PR